### PR TITLE
{devel,lib,system}[GCCcore/13.2.0] ZeroMQ v4.3.5, OpenPGM v5.2.122, libsodium v1.0.19

### PR DESCRIPTION
**Pull request by Micket**
(created using `eb --new-pr`)
- [x] #19053

### CODE REVIEW

File: easybuild/easyconfigs/l/libsodium/libsodium-1.0.19-GCCcore-13.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+toolchain = {'name': 'GCCcore', 'version': '13.2.0'}
+toolchainopts = {'pic': True}
+
+source_urls = [
+    'https://download.%(name)s.org/%(name)s/releases/',
+    'https://download.%(name)s.org/%(name)s/releases/old/',
+    'https://download.%(name)s.org/%(name)s/releases/old/unsupported/',
+]
+sources = [SOURCE_TAR_GZ]
+checksums = ['6f504490b342a4f8a4c4a02fc9b866cbef8622d5df4e5452b46be121e46636c1']
```
Comment (SebastianAchilles) [inline easybuild/easyconfigs/l/libsodium/libsodium-1.0.19-GCCcore-13.2.0.eb:21]:
```suggestion
checksums = ['018d79fe0a045cca07331d37bd0cb57b2e838c51bc48fd837a1472e50068bbea']
```

**Review by SebastianAchilles — CHANGES_REQUESTED** [review]

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in total)
alvis-c1 - Linux Rocky Linux 8.8, x86_64, Intel Xeon Processor (Skylake), Python 3.6.8
See https://gist.github.com/Micket/32befba8ca53f078b9990f610fa3b8dd for a full test report.

**Comment by SebastianAchilles** [comment]
Test report by @sebastianachilles
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in total)
zen2-ubuntu-2310 - Linux Ubuntu 23.10 (Mantic Minotaur), x86_64, AMD EPYC 7452 32-Core Processor (zen2), Python 3.11.6
See https://gist.github.com/SebastianAchilles/a85465746d9ee61fc1da5ab9e0ce84b4 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in total)
node3100.skitty.os - Linux RHEL 8.8, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/boegel/3ea8a60e6ebb14dc2dc88456d441b74e for a full test report.

**Review by boegel — APPROVED** [review]
lgtm

**Comment by boegel** [comment]
Going in, thanks @Micket!

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
@boegelbot please test @ jsc-zen2

**Pull request merged**